README.md

```diff
@@ -1,11 +1,17 @@
 # project-jdp-2108-01
-Projekt zespołowy Kodilla 08.2021
 
-PM Marcin Szuppe
+## Projekt zespołowy Kodilla 08.2021
 
-Kamil Dawid
-Patryk Ziemniak
-Anna Dyrduł
-Julia Sidor
-Dominik Bernatowicz
-Kamil Król
+**Project Manager** Marcin Szuppe
+
+**Contributors**
+- Kamil Dawid
+- Patryk Ziemniak
+- Anna Dyrduł
+- Julia Sidor
```

Comment:
Jej już nie ma ;)

Reply:
Tak to wydaję mi się git

Reply:
Lista uczestników była już wcześniej, ja ją tylko sformatowałem. Poza tym ona też stworzyła kod i mimo że nie była do końca to jednak powinna być wypisana.